srcpkgs/rpi-firmware/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'rpi-firmware'
-_githash="f293685f683c48b1872beeb38c2f7da1f46141a0"
+_githash="8f13114b9ea29bd004151d4a8afa500b2df721be"
 _gitshort="${_githash:0:7}"
 
 pkgname=rpi-firmware
-version=20200819
-revision=2
+version=20201123
+revision=1
 archs="armv6l* armv7l* aarch64*"
 wrksrc="firmware-${_githash}"
+depends="rpi-firmware-network"
 short_desc="Firmware files for the Raspberry Pi (git ${_gitshort})"
-maintainer="Peter Bui <pbui@github.bx612.space>"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
I adopt it, thanks your work

Reply:
Excellent, do you want me to change the maintainer line in this commit or will you do it later?

Reply:
please do it